submissions/LGeorge007/DOM-practice/script.js

```diff
@@ -0,0 +1,261 @@
+        },
+        strenghts: {
+            title: "Сильные стороны",
+            text: "Среди улучшенных юнитов 7-го уровня: Самый высокий показатель атаки. Самый высокий показатель защиты. Второй по величине показатель скорости (выше лишь у фениксов). Второй по величине показатель урона (выше лишь у титанов). Отсутствие разброса урона. Воскрешение союзников. Как следствие: Самая высокая стоимость. Крайне дорогая отстройка Улучшенного Портала Славы ."
+        },
+        weaknesses: {
+            title: "Слабые стороны",
+            text: "Отсутствие магического иммунитета с учётом стоимости найма и среднего здоровья."
+        },
+        img: "img/archangel/archangel.jpg"
+    }
+];
+
+const render = function (item) {
+
+        content.innerHTML = "";
+        const unit = listOfUnits.find(unit => unit.id == item);
+        let templateCard = `
+            <h1>"${unit.description.titleRus} (ориг.${unit.description.titleEng})"</h1>
+            <img src=${unit.img} class="heroImg">
+            <h2>Описание</h2>
+            <p>${unit.description.text}</p>
+            <h2>Особенности</h2>
+            <p>${unit.features.text}</p>
+            <h2>Тактика</h2>
+            <p>${unit.tactics.text}</p>
+            <h2>Сильные стороны</h2>
+            <p>${unit.strenghts.text}</p>
+            <h2>Слабые стороны</h2>
+            <p>${unit.weaknesses.text}</p>`;
+        content.innerHTML = templateCard;
+
+};
+
+const slideMenu = function() {
+
+    navPanel.classList.toggle("nav-toggle");
+    content.classList.toggle("content-toggle");
+    button.classList.toggle("menu-button-toggle");
+
+}
+
+
+
+const attachClickOnMenu = function(event) {
+
+    let target = event.target;
+    if ((target.id !== lastChoice.id) & target.id !=="") {
+        lastChoice.classList.remove("selectedItem");
+        lastChoice = event.target;
+        lastChoice.classList.add("selectedItem");
+        render(target.id);
+        window.scrollTo(0, 0);
+        slideMenu();
+    };
+
+}
+
+const attachClickOnButton = function(event) {
+
+    slideMenu();
+
+}
+
+const loadMenu = function () {
+
+    for (let unit of listOfUnits) {
+        let item = document.createElement("li");
+        item.setAttribute("id", unit.id);
+        item.textContent = unit.description.titleRus;
+        menu.appendChild(item);
+    }
+    navPanel.appendChild(menu);
+    lastChoice = menu.children[startMenuPosition];
+    lastChoice.classList.add("selectedItem");
+
+};
+
+
+const init = function () {
+
+    loadMenu();
+    render(lastChoice.id);
+    menu.addEventListener("click", attachClickOnMenu);
+    button.addEventListener("click", attachClickOnButton);
```

Comment:
You can call `slideMenu` as callback function instead of wrapping it by another function.

Reply:
I agree. 
In the begin, there were 3 functions. Then I wrapped it in one slideMenu. 
But why use another function for slideMenu when it can be called immediately as a callback? 
Fixed it.